src/material-experimental/mdc-progress-spinner/progress-spinner.html

```diff
@@ -0,0 +1,37 @@
+<ng-template #circle>
+  <svg [attr.viewBox]="_viewBox()" class="mdc-circular-progress__indeterminate-circle-graphic"
+       xmlns="http://www.w3.org/2000/svg" focusable="false">
+    <circle [attr.r]="_circleRadius()"
+            [style.stroke-dasharray.px]="_strokeCircumference()"
+            [style.stroke-dashoffset.px]="_strokeCircumference() / 2"
+            [style.stroke-width.%]="_circleStrokeWidth()"
+            cx="50%" cy="50%"/>
+  </svg>
+</ng-template>
+
+<div class="mdc-circular-progress" role="progressbar" #spinnerRoot>
```

Comment:
Is there a reason we couldn't just get rid of this div and put these attributes on the host element?

Reply:
+1, I missed this in the first pass. I would prefer always having the aria stuff on the host element